modules/drivers/sqlserver/resources/metabase-plugin.yaml

```diff
@@ -11,8 +11,7 @@ driver:
     - host
     - merge:
         - port
-        - default: 1433
-          description: Leave empty to use Dynamic Ports, or input specific port. Standard port is 1433.
+        - description: Leave empty to use Dynamic Ports, or input specific port. Standard port is 1433, which needs to be inputted.
```

Comment:
`inputted` sounds a little weird to me here. I think the previous wording was actually fairly clear.